### PR TITLE
Refactor libInfo struct

### PR DESCRIPTION
**Pull request by Meakk**

### CODE REVIEW

File: library/src/engine.cxx [hunk 1]
```diff
@@ -290,29 +290,26 @@ engine::libInformation engine::getLibInfo()
...
-  tmp = "ON";
+  libInfo.Modules["OpenEXR"] = true;
 #else
-  tmp = "OFF";
+  libInfo.Modules["OpenEXR"] = false;
 #endif
-  libInfo.OpenEXRModule = tmp;
 
+  // First version of VTK including the version check (and the feature used)
+#if VTK_VERSION_NUMBER >= VTK_VERSION_CHECK(9, 0, 20200527)
```
Comment (mwestphal) [inline library/src/engine.cxx:312]: how did that compiled before this change ?

**Review by mwestphal — COMMENTED** [review]
a comment

File: library/src/engine.cxx (hunk 1 above)
Comment (Meakk) [reply library/src/engine.cxx]: that's a mistake on my side, you removed that with the drop of VTK 9.0
fixed

Comment (mwestphal) [reply library/src/engine.cxx]: Ha right

**Review by mwestphal — APPROVED** [review]

**Pull request merged**